test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -344,6 +344,11 @@ protected final void beforeInternal() throws Exception {
         cluster().beforeTest(random(), getPerTestTransportClientRatio());
         cluster().wipe(excludeTemplates());
         randomIndexTemplate();
+        customBeforeClass();
```

Comment:
why are you extending this functionality? Just use the junit `@BeforeClass` annotation, see e.g. `TransportMasterNodeActionTests`